mne/inverse_sparse/mxne_optim.py

```diff
@@ -670,10 +673,361 @@ def __call__(self, z):
 
 
 @verbose
-def tf_mixed_norm_solver(M, G, alpha_space, alpha_time, wsize=64, tstep=4,
-                         n_orient=1, maxit=200, tol=1e-8, log_objective=True,
-                         lipschitz_constant=None, debias=True, verbose=None):
-    """Solves TF L21+L1 inverse solver
+def norm_l21_tf(Z, shape, n_orient):
+    if Z.shape[0]:
+        Z2 = Z.reshape(*shape)
+        l21_norm = np.sqrt(stft_norm2(Z2).reshape(-1, n_orient).sum(axis=1))
+        l21_norm = l21_norm.sum()
+    else:
+        l21_norm = 0.
+    return l21_norm
+
+
+@verbose
```

Comment:
no reason to have the `verbose` decorator on a function with no `verbose` parameter, remove